Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM rust:1.77.0 as builder
+LABEL Description="OpenLan CGW (Build) environment"
+
+WORKDIR /usr/src/openlan-cgw
+
+RUN rustup target add x86_64-unknown-linux-gnu
+
+RUN apt-get update -q -y  && apt-get install -q -y \
+	pkg-config \
+	build-essential \
+	cmake \
+	protobuf-compiler \
+	libssl-dev
+
+CMD ["make", "-C", "/usr/src/openlan-cgw", "cgw-app"]
+
+FROM busybox:glibc as cgw-img
+COPY --from=builder /lib/x86_64-linux-gnu/ /lib/x86_64-linux-gnu/
+COPY output/bin/ucentral-cgw /usr/local/bin/ucentral-cgw
+CMD ["ucentral-cgw"]
```

Comment:
It is good practice to setup non-root user to run the application in the container.

Reply:
--disabled-password \
    --gecos "" \
    --home "/nonexistent" \
    --shell "/sbin/nologin" \
    --no-create-home \
    --uid "${UID}" \
    appuser
USER appuser

Reply:
We can track your comments in the issues section as well
Maybe something like
#Improvement - dockerfile should utilize non-root user by default